hostfactory/1.1/providerplugins/azurecc/scripts/src/cyclecloud_provider.py

```diff
@@ -783,25 +729,22 @@ def _create_status(self, input_json, json_writer=None):
                 if not requests_store[request_id].get("lastNumNodes") :
                     requests_store[request_id]["lastNumNodes"] = actual_machine_cnt
                 if actual_machine_cnt < requests_store[request_id]["lastNumNodes"]:
-                    logger.debug("Actual machine count %s is less than previous machines %s", actual_machine_cnt, requests_store[request_id]["lastNumNodes"])
                     request_envelope = self.capacity_tracker.get_request(request_id)
                     if not request_envelope:
-                        logger.warning("Out-of-capacity detected")
-                        logger.warning("No request envelope found for request id %s", request_id)
+                        logger.warning("No request envelope maybe all buckets have hit capacity issue")
```

Comment:
Small tweak in wording
No request envelope found - it may be that all buckets have hit capacity issues?

Reply:
yes it can be updated since we are not sure if buckets have hit capacity issue